src/components/editions/grid/Grid.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import { css } from '@emotion/core';
+import { border, remSpace } from '@guardian/src-foundations';
+import { from } from '@guardian/src-foundations/mq';
+import Byline from 'components/editions/byline';
+import HeaderMedia from 'components/editions/headerMedia';
+import Headline from 'components/editions/headline';
+import Standfirst from 'components/editions/standfirst';
+import type { FC } from 'react';
+import { interview, article as item } from '../../../fixtures/item';
+import Lines from '../lines';
+import { Column } from './Column';
+import Container from './Container';
+import Grid from './Grid';
+
+const video = {
+	kind: 0,
+	value: {
+		kind: 1,
+		video: {
+			posterUrl:
+				'https://media.guim.co.uk/032aa99755664104fbfc4cbe45cfae0243dce462/0_0_3972_2234/master/3972.jpg',
+			videoId: 'wD_bWOEuuoc',
+			duration: 59,
+			atomId: '26401ff7-24d0-4ba5-8882-2c32c2b379f0',
+			title:
+				'Super Bowl LV: Tom Brady MVP as Buccaneers beat Chiefs 31-9 – video report',
+		},
+	},
+};
+
+const borderStyles = css`
+	${from.tablet} {
+		border-right: 1px solid ${border.secondary};
+		padding-right: ${remSpace[3]};
+	}
+`;
+
+const lineStyles = css`
+	${from.tablet} {
+		margin-right: -${remSpace[3]};
+	}
+`;
+
+const Article: FC = () => (
+	<Container>
+		<Grid smCols={12} mdCols={12} lgCols={12}>
```

Comment:
I'm aware this is just an example but should it be `<Grid smCols={12} mdCols={8} lgCols={8}>` otherwise it will look like the fullwidth article?

Reply:
So these are just the numbers of columns in each row, they should only be different if the designs reflect that.

Ie if on mobile the layout is 4 cols wide then the value should be 4. Its on the `Column` component that we set the widths.

Does that make sense?